lib/kafka_ex/server.ex

```diff
@@ -932,6 +933,10 @@ defmodule KafkaEx.Server do
       defp config_sync_timeout(timeout \\ nil) do
         timeout || Application.get_env(:kafka_ex, :sync_timeout, @sync_timeout)
       end
+
+      defp default_partitioner() do
```

Comment:
I think that the extra parenthesis here will fail in credo checking, which will fail the CI tests. You can check credo locally by running `mix credo`

Reply:
No, it won't. but I can fix it to align with the project style.